src/content/blog/ddev-and-xz-backdoor.md

```diff
@@ -0,0 +1,67 @@
+
+    ```
+    $ ddev exec -s db xz --version
+    xz (XZ Utils) 5.2.5
+    liblzma 5.2.5
+    ```
+* All distributed DDEV images were built before the episode began and were not rebuilt during the episode.
+* In its Golang code, DDEV uses [ulikunitz/xz](https://github.com/ulikunitz/xz), which is quite stable and unrelated to the release in question. DDEV includes ulikunitz/xz 0.5.11, the most recent stable version, from December 2022.
+
+## How can I find out if I have an affected `xz` version?
+
+In general, the affected versions of `xz` were 5.6.0 and 5.6.1. 
+
+`xz --version` will tell you about your host machine, and `ddev exec -s db xz --version` will tell you about `xz` in the DDEV database container.
+
+## What is `xz` anyway and how does DDEV use `xz`?
+
+`xz` is a powerful compression tool, like `gzip` but with more effective compression.
+
+DDEV uses `xz` so that it can read or create `xz`-compressed database dumps, and so it can read `xz`-compressed files tarballs. The significant and long-term DDEV sponsor [B13](https://b13.com/) sponsored this feature [in 2022](https://github.com/ddev/ddev/pull/3721) 🙏🏼.
+
+## How can I fix my host computer if I have an affected `xz` version?
+
+Unless you use Homebrew, you probably don't have the affected version on your host computer, and Homebrew users are probably not affected. However, `brew upgrade` will downgrade `xz` to 5.4.6. Arch Linux and derived distros need to upgrade to get 5.6.1-2, which is built from source.
+
+## What does this mean for small open-source projects like DDEV?
+
+For me, the story here isn't about the vulnerability that didn't make it, but about the open-source story behind it. A small project with an overworked maintainer and insufficient community support got a new maintainer who spent two years gaining trust and then essentially took over and released code that had been carefully placed over that whole time. We all know that we depend on hundreds of these projects. 
+
+But one of these projects. You know that we've been working on [building our maintainer team](https://ddev.com/blog/expanding-ddev-maintainer-team) and [training contributors](https://ddev.com/blog/contributor-training/). We absolutely need to continue **building DDEV's financial stability** and **train contributors and maintainers**. Please:
+
+* Join us in [financially supporting the project](https://github.com/sponsors/ddev). You or your company can support us directly (we invoice for support, accept funds many ways) or you can support via GitHub Sponsors.
+* Join us as we restart the Contributor Training in coming months. We need lots of competent eyeballs on the project.
+* Take security seriously! Pay attention. Although DDEV may seem to be too small for an attacker to take time with, it probably isn't. Imagine if someone were to be able to plant something that affected your computer or your project code. It might be worth it to attach DDEV. 
```

Comment:
```suggestion
* Take security seriously! Pay attention. Although DDEV may seem to be too small for an attacker to take time with, it probably isn't. Imagine if someone is able to plant something that affects your project code or your computer in general. It might be worth it to attach DDEV. 
```

"to attach DDEV"? attaching it to what?